src/components/ui/Animation.js

```diff
@@ -256,31 +253,36 @@ CIQ.Animation = function (stx, animationParameters, easeMachine) {
     });
 
     stx.append('draw', function () {
-        if (filterSession) { return; }
-        if (this.chart.dataSet && this.chart.dataSet.length && this.mainSeriesRenderer && this.mainSeriesRenderer.supportsAnimation) {
-            if (flashingColorThrottleCounter % flashingColorThrottle === 0) {
-                flashingColorIndex++;
-                flashingColorThrottleCounter = 0;
-            }
-            flashingColorThrottleCounter++;
+        if (!currentSpotElement) {
+            currentSpotElement = document.createElement('span');
+            currentSpotElement.className = 'cq-spot';
+            this.chart.panel.holder.parentElement.appendChild(currentSpotElement);
+        }
+
+        currentSpotElement.style.display = 'none';
 
+        if (filterSession) { return; }
+        if (
+            this.layout.timeUnit === 'second'
```

Comment:
force animation to just display when then the interval is `1 second`

Reply:
... wait. Why?

Reply:
@brucebinary think it's not meaningful if animation exist when the interval is something but `1 second`